cli/module_build.go

```diff
@@ -255,6 +255,39 @@ func ModuleBuildLogsAction(c *cli.Context) error {
 	return nil
 }
 
+// ModuleBuildLinkRepoAction links a github repo to your module.
+func ModuleBuildLinkRepoAction(c *cli.Context) error {
+	linkID := c.String(moduleBuildFlagOAuthLink)
+	moduleID := c.String(moduleFlagPath)
+	repo := c.String(moduleBuildFlagRepo)
+
+	req := buildpb.LinkRepoRequest{
+		Link: &buildpb.RepoLink{
+			OauthAppLinkId: linkID,
+			Repo:           repo,
+		},
+	}
+	var found bool
+	req.Link.OrgId, req.Link.ModuleName, found = strings.Cut(moduleID, ":")
+	if !found {
+		return fmt.Errorf("the given module ID '%s' isn't of the form org:name", moduleID)
+	}
```

Comment:
I suggest you use:
```go
modID, err := parseModuleID(moduleID)
org, err := getOrgByModuleIDPrefix(client, modID.prefix)
req.Link.OrgID = org.id
req.Link.ModuleName = modID.name
```

(see module_registry.go)

Otherwise users will have to pass the id as `orgid:name` instead of the much more convenient `namespace:name`